<ui version="4.0" >
 <class>InstallBootloaderFrm</class>
 <widget class="QDialog" name="InstallBootloaderFrm" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Install Bootloader</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="6" row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="rbutilqt.qrc" >:/icons/icons/wizard.xpm</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Select your device in the filesystem</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="lineMountPoint" />
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="buttonBrowse" >
     <property name="text" >
      <string>&amp;Browse</string>
     </property>
     <property name="icon" >
      <iconset resource="rbutilqt.qrc" >:/icons/icons/system-search.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Download and select an original Firmware</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QLineEdit" name="lineOriginalFirmware" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>select the original Firmware</string>
     </property>
     <property name="accessibleDescription" >
      <string>select the original Firmware</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" >
    <widget class="QPushButton" name="buttonBrowseOF" >
     <property name="text" >
      <string>&amp;Browse</string>
     </property>
     <property name="icon" >
      <iconset resource="rbutilqt.qrc" >:/icons/icons/system-search.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="buttonOk_2" >
       <property name="text" >
        <string>&amp;Ok</string>
       </property>
       <property name="icon" >
        <iconset resource="rbutilqt.qrc" >:/icons/icons/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel_2" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="rbutilqt.qrc" >:/icons/icons/process-stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rbutilqt.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonOk_2</sender>
   <signal>clicked()</signal>
   <receiver>InstallBootloaderFrm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>476</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel" >
     <x>531</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel_2</sender>
   <signal>clicked()</signal>
   <receiver>InstallBootloaderFrm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>556</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel" >
     <x>599</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
